docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -0,0 +1,359 @@
+    - ufs-community/UFS_UTILS/pull/<pr_number>
+    - ufs-community/ufs-weather-model/pull/<pr_number>
+
+    ## DOCUMENTATION:
+    If this PR is contributing new capabilities that need to be documented, please also 
+    include updates to the RST files (docs/UsersGuide/source) as supporting material.
+
+    ## ISSUE (optional): 
+    If this PR is resolving or referencing one or more issues, in this repository or 
+    elewhere, list them here. For example, "Fixes issue mentioned in #123" or "Related to 
+    bug in https://github.com/ufs-community/other_repository/pull/63"
+
+    ## CONTRIBUTORS (optional): 
+    If others have contributed to this work aside from the PR author, list them here
+
+
+Additional Guidance
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+**TITLE:** Titles should give code reviewers a clear idea of what the change will do in approximately 5-10 words. Some good examples from the past:
+
+    * Make thompson_mynn_lam3km ccpp suite available
+    * Fix module loads on Hera
+    * Add support for Rocoto with generic LINUX platform
+
+All of the above examples concisely describe the changes contained in the pull request. The title will not get cut off in emails and web pages. In contrast, here are some made-up (but plausible) examples of BAD pull request titles:
+
+    * Bug fixes (Bug fixes on what part of the code?)
+    * Changes to surface scheme (What kind of changes? What surface scheme?)
+
+**DESCRIPTION OF CHANGES:** The first line of the description should be a single-line "purpose" for this change. Note the type of change (i.e., bug fix, feature, enhancement, textonly). Summarize the problem, proposed solution, and required changes. If this is an enhancement or new feature, describe why the change is important.
+
+**DOCUMENTATION:** Developers should include documentation on new capabilities and enhancements by updating the appropriate ``.rst`` documentation files in their ``ufs-srweather-app`` fork prior to the PR. These documentation updates should be noted in the "Documentation" section of the PR message. If necessary, contributors may submit the ``.rst`` documentation in a subsequent PR. In these cases, the developers should include any existing documentation in the "Documentation" section of the initial PR message or as a file attachment to the PR. Then, the contributor should open an issue (see :numref:`Step %s <Issue>`) reflecting the need for official ``.rst`` documentation updates and include the issue number and explanation in the "Documentation" section of the initial PR template.
+ 
+
+Tips, Best Practices, and Protocols to Follow When Issuing a PR
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Indicate urgency.** If a PR is particularly urgent, this information should be provided in the PR "Description" section, and multiple code management team members should be tagged to draw attention to this PR. After submitting the PR, a "high priority" label should be added to it. 
```

Comment:
I think making a note to add labels in general would be really good. This could be a separate bullet but wanted to comment on it here since you mentioned adding high priority in this bullet.

Reply:
Never mind - see this below!